third-party/realdds/include/realdds/dds-option.h

```diff
@@ -0,0 +1,53 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2022 Intel Corporation. All Rights Reserved.
+
+#pragma once
+
+
+#include <third-party/json_fwd.hpp>
+
+#include <string>
+
+
+namespace realdds {
+
+
+struct dds_option_range
+{
+    float min;
+    float max;
+    float step;
+    float default_value;
+};
+
+class dds_option
+{
+    std::string _name;
+    float _value;
+    dds_option_range _range;
+    std::string _description;
+
+    dds_option( nlohmann::json const & j );
+
+public:
+    dds_option() = default;
```

Comment:
Why do we need this? I imagine the minimum ever would get an option name, but even then it would be protected and overwritten?

Also, `_value`, `_range` are uninitialized. Aren't you getting a warning?

Reply:
I did not want to create a constructor with many parameters, especially when you can set the values later
(rs-dds-server uses this to create an option and then set values it gets from LibRS interface)

When I started I didn't use option name, only added later and I agree it can be a minimum.
(I still need to consider your comment about device options for now I also think we will need stream name if using stream level options)